### PR TITLE
Hw10

### PR DESCRIPTION
**Pull request by bohdank3**

### CODE REVIEW

File: hw10/src/FileNavigator.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+mport java.util.*;
```
Comment (serhii-tokranov) [inline hw10/src/FileNavigator.java:1]: This is a rude mistake!

File: hw10/src/FileNavigator.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                if (fileData.size <= maxSize) {
+                    result.add(fileData);
+                }
+            }
+        }
+        return result;
+    }
+
+    /*5. Реалізувати метод remove у класі FileNavigator. Метод видаляє шлях і пов'язані з ним файли, виходячи з значення шляху, переданого як параметр.*/
+    public void remove(String path) {
```
Comment (serhii-tokranov) [inline hw10/src/FileNavigator.java:48]: It is not a good idea to message a user that something happened (the path was removed) before the action (`files.remove(path)`) was executed.

File: hw10/src/FileNavigator.java [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        if (files.containsKey(path)) {
+            System.out.println(files.get(path).size()+" элементов удалили с пути "+path);
+        } else {
+            System.out.println("не нашли файлов по этому пути "+path);
+        }
+        files.remove(path);
```
Comment (serhii-tokranov) [inline hw10/src/FileNavigator.java:54]: If `path` does not exist - why do we call the remove method?

**Comment by bohdank3** [comment]
Hw10

**Pull request merged**